src/resin-event-log.js

```diff
@@ -66,6 +66,13 @@ var EVENTS = {
 		'deactivate',
 		'pinToRelease'
 	],
+	release: [
+		'openAddRelease',
+		'copyInstructions',
+		'clickInstallLink',
+		'clickGettingStarted',
```

Comment:
I'm fine with `deployFromUrl` but I think that the rest should be renamed a bit just to make them similar with the rest of the events. How about:
* addModalOpen
* instructionsCopy
* installLinkClick (balenaCLI?)
* gettingStartedClick

Reply:
I was late :(

Reply:
@thgreasi I can rename them as a bugfix, nothing is using them anyway